proxy/controller-grpc/src/lib.rs

```diff
@@ -0,0 +1,531 @@
+#![allow(dead_code)]
```

Comment:
This `allow(dead_code)` isn't necessary, AFAICT.

Reply:
I reviewed this file by comparing it chunkwise to the original proxy/src/control/pb.rs and  match_.rs. This was tedious to do because the ordering of the chunks was not preserved.

Is the new version of the source code in the preferred order now or are we going to re-order this code further? If there's some better order then let's reorder it now.

Reply:
the reordering wasn't intentional.  From what I could tell, the original ordering wasn't particularly intentional either.  I'm open to input.

Reply:
I don't have any strong feelings about the ordering. I just want to point out for future reference that it would make the reviews go easier if we can avoid spurious changes to the ordering. However, in this case I did the review of the reordered code so there's no need to try to preserve the original order unless we think the original order is better. (I've no opinion; we should develop guidelines about how to organize code like this.)